.github/ISSUE_TEMPLATE.md

```diff
@@ -3,11 +3,11 @@ Before submitting, please read over our commonly reported issues to prevent dupl
 
 ### All common issues
 
-* [common issues](/docs/common-issues)
+* [common issues](/docs/common-issues.md)
```

Comment:
@hanumanthan I think you misunderstood my comment - if you try to open a new issue, and click preview, and try to click one of the links, you will see that it goes to the wrong place.

In other words, I think this needs to be `../blob/master/docs/common-issues.md` in order for it to work properly.